docs/experience/browser-memory.md

```diff
@@ -37,4 +37,4 @@ Sematext Experience [Browser SDK](https://sematext.com/docs/agents/browser/) use
 />
 
 ## Supported Browsers
-At the moment, the only web browser that supports memory usage measurements is Chrome starting from version **88**. Until that version, you can register your website for [Origin Trials](https://developers.chrome.com/origintrials/#/view_trial/1281274093986906113) or turn on the **#experimental-web-platform-features** flag in Chrome. 
+At the moment, the only web browser that supports memory usage measurements is Chrome starting from version **88**. To collect memory usage from pre-88 Chrome versions you can register your website for [Origin Trials](https://developers.chrome.com/origintrials/#/view_trial/1281274093986906113) or turn on the **#experimental-web-platform-features** flag in Chrome. 
```

Comment:
The point I was trying to get across that tuning on the flag in *your* Chrome will only collect *your* Chrome's memory usage.  That's very different from registering for Origin Trials, which is about collecting memory usage from *other people's* Chrome browsers (that have that same feature enabled).

Reply:
Let me remove the `or turn on the **#experimental-web-platform-features** flag in Chrome` from the above text. It can be confusing as you are only controlling your own browser, not other people browsers .